homework10/App.java

```diff
@@ -0,0 +1,26 @@
+package homework10;
+
+import homework10.configuration.FileLoggerConfiguration;
+import homework10.configuration.FileLoggerConfigurationLoader;
+import homework10.enums.LoggingLevel;
+import homework10.exceptions.FileMaxSizeReachedException;
+
+import java.io.IOException;
+
+public class App {
+    public static void main(String[] args){
+        FileLoggerConfiguration fileLoggerConfiguration =
+                new FileLoggerConfiguration("src/main/resources/log.txt", LoggingLevel.INFO, 130, "txt");

```

Comment:
Do NOT hardcode the path `"src/main/resources/"`! Just provide the filename. Also, it seems that you're duplicating the file format - `log.txt` and the format value `txt`. Fix it.